crates/wadm/src/scaler/configscaler.rs

```diff
@@ -36,6 +36,14 @@ impl<C: ConfigSource + Send + Sync + Clone> Scaler for ConfigScaler<C> {
         &self.id
     }
 
+    fn kind(&self) -> &str {
+        "ConfigScaler"
```

Comment:
Should this be a const too?

Reply:
Yeah const in the file works, I tried to make this a const in the trait but with our backoff wrapping we can't proxy it down to the scalers